src/elements/image.js

```diff
@@ -78,15 +81,18 @@ class Image extends Component {
 }
 
 const asEnums = [ 'div', 'img' ];
+const typeEnums = [ 'person', 'user' ];
 
 Image.propTypes = {
     as: PropTypes.oneOf(asEnums),
-    avatar: PropTypes.bool,
+    border: PropTypes.number,
```

Comment:
Correct me If I am wrong, I have been using border props like this
`const borderStyle = border ? (borderInverse ? `${border}px solid #fff` : `${border}px solid #dbe0e3`) : 'none';`
Currently, passing border boolean with value set true won't work because of this `${border}px solid #fff`. Should I make border width 1px when border props is set to true ??

Reply:
Fixed.